src/Compilers/Core/Desktop/AnalyzerFileReference.cs

```diff
@@ -208,13 +213,12 @@ internal void AddAnalyzers(ImmutableArray<DiagnosticAnalyzer>.Builder builder, s
                 // If there are no analyzers, don't load the assembly at all.
                 if (!analyzerTypeNameMap.ContainsKey(language))
                 {
-                    this.AnalyzerLoadFailed?.Invoke(this, new AnalyzerLoadFailureEventArgs(AnalyzerLoadFailureEventArgs.FailureErrorCode.NoAnalyzers, null, null));
```

Comment:
@tmeschter You are right, the place where that diagnostic is generated is here: http://source.roslyn.io/#Microsoft.CodeAnalysis.Desktop/AnalyzerFileReference.cs,266 which is during the call to ` GetLanguageSpecificAnalyzers`
I take my original comment back, and :+1: for this change.